translations/get5.phrases.txt

```diff
@@ -0,0 +1,265 @@
+"Phrases"
+{
+    "ReadyToVetoInfoMessage"
+    {
+        "en"            "Type {GREEN}!ready {NORMAL}when your team is ready to veto."
+    }
+    "WaitingForCastersReadyInfoMessage"
+    {
+        "en"            "Waiting for the casters to type {GREEN}!ready {NORMAL}to begin."
+    }
+    "ReadyToRestoreBackupInfoMessage"
+    {
+        "en"            "Type {GREEN}!ready {NORMAL}when your team is ready to restore the match backup."
+    }
+    "ReadyToKnifeInfoMessage"
+    {
+        "en"            "Type {GREEN}!ready {NORMAL}when your team is ready to knife."
+    }
+    "ReadyToStartInfoMessage"
+    {
+        "en"            "Type {GREEN}!ready {NORMAL}when your team is ready to begin."
+    }
+    "WaitingForEnemySwapInfoMessage"
+    {
+        "#format"       "{1:s}"
+        "en"            "{1} won the knife round. Waiting for them to type !stay or !swap."
+    }
+    "WaitingForGOTVBrodcastEndingInfoMessage"
+    {
+        "en"            "The map will change once the GOTV broadcast has ended."
+    }
+    "NoMatchSetupInfoMessage"
+    {
+        "en"			"There is no match setup"
+    }
+    "YourAreNotAPlayerInfoMessage"
+    {
+        "en"            "You are not a player in this match"
+    }
+    "TeamIsFullInfoMessage"
+    {
+        "en"            "Your team is full"
+    }
+    "TeamForfeitInfoMessage"
+    {
+        "#format"       "{1:s}"
+        "en"            "{1} failed to ready up in time and has forfeit."
+    }
+    "MinutesToForfeitMessage"
+    {
+        "#format"       "{1:s},{2:d}"
+        "en"            "{1} has {2} minutes left to ready up or they will forfeit the match."
+    }
+    "SecondsToForfeitInfoMessage"
+    {
+        "#format"       "{1:s},{2:d}"
+        "en"            "{1} has {2} seconds left to ready up or they will forfeit the match."
+    }
+    "%s has 10 seconds to ready up or forfeit the match."
+    {
+        "#format"       "{1:s}"
+        "en"            "{1} has 10 seconds to ready up or forfeit the match."
+    }
+    "PausePeriodSuffix"
+    {
+        "en"            "for this half"
+    }
+    "MaxPausesUsedInfoMessage"
+    {
+        "#format"       "{1:d},{2:s}"
+        "en"            "Your team has already used the max {1} pauses{2}."
+    }
+    "MaxPausesTimeUsedInfoMessage"
+    {
+        "#format"       "{1:d},{2:s}"
+        "en"            "Your team has already used the max {1} seconds of pause time{2}."
+    }
+    "MatchPausedByTeamMessage"
+    {
+        "en"            "%N paused the match."
```

Comment:
Official documentation confused me: in this docs: https://wiki.alliedmods.net/Translations_(SourceMod_Scripting)#File_Format author says that "only the following types are currently allowed:" but there is no %N format here.
But I've changed this string as any other formats and it works perfect when I test it.

And now I made fix for this string too.